client/src/components/commons/ReviewCard.jsx

```diff
@@ -26,6 +25,7 @@ const {
   commentTime,
   textTime,
 } = styles;
+import Shows from './../showsView/Shows';
```

Comment:
Do not import another component if it's not being used in the current file.

Reply:
removed